draft-dekater-scion-dataplane.md

```diff
@@ -250,6 +254,24 @@ The full forwarding process for a packet transiting an intermediate AS consists
 **Note:** The current SCION implementation uses the UDP/IP protocol as underlay. However, the use of other underlay protocols is possible and allowed.
 
 
+### Configuration
+
+Border routers require mappings from SCION  interface IDs to underlay addresses. Such information must be supplied to each router in an out of band fashion (e.g in a configuration file). For each link to a neighbor, these values must be configured:
+
+- Local interface ID
+- Neighbor type (core, parent, child, peer), depending on link type (see [](#paths-links)). Link type depends on mutual agreements between the organizations operating the ASes at each end of each link.
+- Neighbor ISD-AS number
+- Neighbor interface underlay address
+
+In addition, each router must be configured with a list of interface IDs in use within the AS, containing:
+
+- Interface ID
+- "intra-protocol" address of the remote router
+
+In order to forward traffic to internal endpoints (traffic with interface ID 0), border routers need to associate endpoint addresses with concrete host addresses of the internal AS network. The nature of internal host addresses and the translation method are not specified by this document. They only depend on implementation and the choices of each AS's administrator. In current practice, internal host addresses are IP addresses (V6 or V4), both of which can be used as endpoint addresses in SCION headers; thereby requiring no translation.
```

Comment:
Does this fit better in previous section  Intra-Domain Forwarding Process ?

Reply:
I'd rather address the configuration topic in one place.

Reply:
Rearanged as per our discussion.